### ISSUE TITLE
Some other questions regarding Microhouse_WRENchassis_v0.5.dxf

### ISSUE THREAD

**Issue by skovholt** [wikihouseproject/Microhouse#8]
Hi,

some additional questions regarding Microhouse_WRENchassis_v0.5.dxf:

1
Single puzzle joints in spacer components: when i put these 0.5mm appart there seems to be some extra spacing on the sides (about 1.9mm). Is it intentional?
![puzzlejoint_spacers](https://cloud.githubusercontent.com/assets/8075211/23164917/e70fafda-f839-11e6-84bb-f60fb4ef5d3a.png)
![puzzlejoint_spacers2](https://cloud.githubusercontent.com/assets/8075211/23164921/eae7c3cc-f839-11e6-96fd-41f0b5bbc3ca.png)

2
FRA-A/S3/EX seems to be missing in the dxf?

3
I guess there should there be dogbones in this location on spacers? Picture is from full scale test of some pieces and not screwed together.

![missing_dogbones2](https://cloud.githubusercontent.com/assets/8075211/23165312/c6315b4a-f83b-11e6-96a4-e0e4a8b826f4.png)
![missing_dogbones3](https://cloud.githubusercontent.com/assets/8075211/23165313/c8bc82fe-f83b-11e6-8e1a-e8e2463da924.png)

**Comment by claytonprest** [comment]
Hi @skovholt 

1. This is the same condition as the dovetail joints in the Fins, there is a small offset of 0.5mm applied the joint before the 0.25mm general in-set on all parts. It's not needed, and shouldn't make much difference. You can choose to leave it as is or reduce it for a tighter fit, it's worth checking how it fits with your material and full-scale tests first.

2, You're right, FRA-A/S3/EX was mistakenly labelled in the .dxf cutting file. I've fixed this now, so should be all correct in the latest version.

3. Yes, there should be 'dog-bone' notches in the internal corners here, and is an error on our part. I've modelled these now, and the most recent update to the .dxf in the Microhouse repo now includes these pockets. If you have already cut parts like above, a quick rub with a file should allow all pieces to fit.

Thanks again for raising these issues. It's super important to have feedback like this from you and others in the community. Hope the rest of your project is going well?

**Comment by skovholt** [comment]
Hi @claytonprest, thanks for your feedback

1. Ok, we left it with the spacing as in the dxf, and it seems to work ok as is. 

3. Yes, I became aware of this a bit into production, so I fixed it on some pieces, the rest we can fix manually.

The project has so far gone quiet smoothly, we'we milled parts for 4 frames/spacers/reinforcers/connectors, with roof, wall and some flooring in 18mm OSB.  We don't have a vacuum table for our cnc, but used composite brad nails with good results. Don't know yet when it will be assembled. The production is commissioned by @twengel (Slack) and has been cnc cut at http://fellesverkstedet.no/ a production space and fablab in Oslo.
